app/src/test/java/com/example/tanya/godeltechchallengeandroid/domain/interactor/GetStartStatusTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.example.tanya.godeltechchallengeandroid.domain.interactor
+
+import com.example.tanya.godeltechchallengeandroid.data.prefs.ApplicationRepository
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+
+
+class GetStartStatusTest {
+
+    @Mock private lateinit var applicationRepository: ApplicationRepository
+    private lateinit var getStartStatus: GetStartStatus
+
+    private val testValue = true
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.initMocks(this)
+        getStartStatus = GetStartStatus(applicationRepository)
+    }
+
+    @Test
+    fun onGetStartStatus_shouldCallApplicationRepository() {
```

Comment:
The test is wrong. It has to test the GetStartStatus instance but I can not see it.